lib/routemaster/responses/future_response.rb

```diff
@@ -45,6 +32,37 @@ def initialize(&block)
       def method_missing(m, *args, &block)
         value.public_send(m, *args, &block)
       end
+
+      def value
+        @future.value.tap do |v|
+          if v.nil? && @future.rejected?
```

Comment:
Why check for the value being nil here? The value will always be nil if the future was rejected according to the docs, so this check confuses me.